nemo/collections/asr/modules/audio_preprocessing.py

```diff
@@ -274,6 +277,7 @@ def __init__(
             nb_max_freq=nb_max_freq,
             stft_exact_pad=stft_exact_pad,  # Deprecated arguments; kept for config compatibility
             stft_conv=stft_conv,  # Deprecated arguments; kept for config compatibility
+            mel_norm=mel_norm,
```

Comment:
`featurizer_class` has two options. You may need to change same in `FilterbankFeaturesTA`.

Reply:
Torchaudio should not be the default for anything. We don't install it and it's not a requirement, but an optional one

Reply:
You'll need to add this argument and basically just ignore it in the TA variant (with logging that you are ignoring it)

Reply:
I added the parameter to the TA variant. Interestingly, `torchaudio.transforms.MelSpectrogram` supports the slaney normalization but (similar to Tensorflow and Kaldi) defaults to not using it.